.gitignore

```diff
@@ -1,7 +1,13 @@
 build
 node_modules
 tmp
-test*.js
-test*.sh
 .sonar*
 .DS_Store
+
+# Test files 
+/src/test
```

Comment:
This allows to create test/debug typescript test files in `src` that aren't tracked by git